src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -2,12 +2,13 @@
 import * as React from 'react';
 import {Text, View, StyleSheet, TouchableOpacity, Pressable, Platform} from 'react-native';
 import ToastMessage from 'react-native-toast-message';
+import CheckBox, {CheckBoxBase} from '@react-native-community/checkbox';
 import TextAreaChat from '../../../components/TextAreaChat';
 import ToggleSwitch from '../../../components/ToggleSwitch';
 import {profileSettingsDMpermission} from '../../../service/profile';
 import {addDotAndRemoveNewline} from '../../../utils/string/TrimString';
 import {COLORS} from '../../../utils/theme';
-import {useDynamicColors} from '../../../hooks/useToggleColors';
+import {PencilIcon} from '../../../assets';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `CheckBoxBase` is not used in the code. It's a good practice to remove unused imports to keep the code clean and maintainable.
```diff
- import CheckBox, {CheckBoxBase} from '@react-native-community/checkbox';
+ import CheckBox from '@react-native-community/checkbox';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->